modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase

```diff
@@ -2989,6 +2989,53 @@ definition {
 		}
 	}
 
+	@description = "LPS-155962 - Verify that it's possible to restrict a previously created Object"
+	@priority = "4"
+	test CanRestrictExistingObject {
+		task ("Add first Custom Object (via API)") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 161897A",
+				objectName = "CustomObject161897A",
+				pluralLabelName = "Custom Objects 161897A");
+		}
+
+		task ("Add second Custom Object (via API)") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 161897B",
+				objectName = "CustomObject161897B",
+				pluralLabelName = "Custom Objects 161897B");
+		}
```

Comment:
Since we're adding two Objects via API with similar names, we can just use a loop here.